### PR TITLE
505 past date validations

### LINKED ISSUE #505: Remove past date validations
After discussion with @tom95 we decided we no longer need these validations since it is unnecessary code overhead and prevents past events to be edited.

### PR DESCRIPTION
**Pull request by bjrne**
Closes #505.

### CODE REVIEW

**Review by ekrebs5 — APPROVED** [review]

File: spec/features/event_spec.rb [hunk 1]
```diff
@@ -135,12 +135,12 @@
       end
     end
 
-    it "should not allow dates in the past" do
+    it "should allow dates in the past" do
```
Comment (querenker) [inline spec/features/event_spec.rb]: I am not sure if this test communicates the right thing. If I were a new developer to this project I would ask what's the purpose of creating events in the past. Allowing old events to edit makes sense, but creating "old" events?

**Review by querenker — CHANGES_REQUESTED** [review]
I think this one test is worth a discussion, but otherwise it looks good to me 👍

File: spec/features/event_spec.rb (hunk 1 above)
Comment (bjrne) [reply spec/features/event_spec.rb]: Wow, didn't think of that. I like the idea, would changing the test action to editing an existing event and saving it without errors be what you suggested? (Did that by now, please look at the new commits 😃 )

**Review by querenker — APPROVED** [review]

File: spec/features/event_spec.rb (hunk 1 above)
Comment (querenker) [reply spec/features/event_spec.rb]: Looks good to me 👍

**Pull request merged**